### PR TITLE
Update layout.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/edb_core/layout.py [hunk 1]
```diff
@@ -90,40 +89,38 @@ def update_primitives(self):
 
     @property
     def primitives(self):
-        """
+        """Retrieve all prmitives.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:92]:
```suggestion
        """Retrieve all primitives.
```

File: pyaedt/edb_core/layout.py [hunk 2]
```diff
@@ -150,13 +147,12 @@ def circles(self):
 
     @property
     def paths(self):
-        """
+        """Retrive all paths.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:150]:
```suggestion
        """Retrieve all paths.
```

File: pyaedt/edb_core/layout.py [hunk 3]
```diff
@@ -166,12 +162,12 @@ def paths(self):
 
     @property
     def bondwires(self):
-        """
+        """Retrive all bondwires.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:165]:
```suggestion
        """Retrieve all bondwires.
```

File: pyaedt/edb_core/layout.py [hunk 4]
```diff
@@ -15,7 +15,7 @@
     warnings.warn('This module requires pythonnet.')
 
 class EdbLayout(object):
-    """EDB Layout class."""
+    """edbLayout class."""
```
Comment (akaszynski) [inline pyaedt/edb_core/layout.py:18]:
```suggestion
    """EdbLayout class."""
```

File: pyaedt/edb_core/layout.py [hunk 5]
```diff
@@ -59,26 +59,25 @@ def db(self):
...
         -------
         dict
-            Dictionary of Layers
+            Dictionary of layers.
         """
         return self._parent.core_stackup.stackup_layers.layers
 
     @aedt_exception_handler
     def update_primitives(self):
         """
```
Comment (akaszynski) [inline pyaedt/edb_core/layout.py:73]: Have the short description on the same line as the beginning of the docstring.

File: pyaedt/edb_core/layout.py [hunk 6]
```diff
@@ -59,26 +59,25 @@ def db(self):
...
-        Update Primitives list from Edb Database
+        Update primitives list from the EDB database.
 
         Returns
         -------
         bool
-            ``True`` if succeeded
+             ``True`` when successful, ``False`` when failed.
```
Comment (akaszynski) [inline pyaedt/edb_core/layout.py:79]:
```suggestion
            ``True`` when successful, ``False`` when failed.
```

File: pyaedt/edb_core/layout.py [hunk 7]
```diff
@@ -300,22 +294,21 @@ def parametrize_polygon(self, polygon,selection_polygon, offset_name="offsetx",
         Parameters
         ----------
         polygon : 
-            Polygon to parametrize.
+            Name of the plygon.
```
Comment (akaszynski) [inline pyaedt/edb_core/layout.py:297]:
```suggestion
            Name of the polygon.
```

**Review by akaszynski — APPROVED** [review]
Tentatively approved with suggestions.  Please have those methods with the @property decorator changed to nouns rather than verbs.

**Pull request merged**